Eventy/src/main/java/org/example/eventy/events/repositories/EventRepository.java

```diff
@@ -49,4 +51,22 @@ List<Event> findOrganizedEventsByUserBetween(
     List<Event> findAttendingEventsByUserBetween(@Param("userId") Long userId,
                                                  @Param("startDate") LocalDate startDate,
                                                  @Param("endDate")LocalDate endDate);
+
+    @Query("SELECT e FROM Event e " +
+            "WHERE (:search = '' OR LOWER(e.name) = LOWER(:search)) " +
```

Comment:
maybe make search for name and description like %search%? you know just like contains?